renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -3112,6 +3112,92 @@ bool WrappedVulkan::Serialise_vkCreateDevice(SerialiserType &ser, VkPhysicalDevi
         CHECK_PHYS_EXT_FEATURE(sampler2DViewOf3D);
       }
       END_PHYS_EXT_CHECK();
+
+      BEGIN_PHYS_EXT_CHECK(VkPhysicalDeviceExtendedDynamicState3FeaturesEXT,
+                           VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_EXTENDED_DYNAMIC_STATE_3_FEATURES_EXT);
+      {
+        // disabling features requiring extensions we don't support
```

Comment:
This code is not necessary and should be removed please. The application will only have enabled features that are reported as supported and this disabling is already handled by patching the return value from the feature query in the first place during capture.